src/rpc/mining.cpp

```diff
@@ -190,13 +190,24 @@ static UniValue generatetoaddress(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Error: Invalid address");
     }
 
-    auto myIDs = pcustomcsview->AmIOperator();
-    if (!myIDs) {
-        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Error: I am not masternode operator");
+    CKeyID passedID = destination.which() == PKHashType ? CKeyID(*boost::get<PKHash>(&destination)) :
```

Comment:
Could you please break this line into a switch statement instead? That would be far more readable and way easier to maintain. Everything else looks good.